### PR TITLE
Modify program

### PR DESCRIPTION
**Pull request by sysim**

### CODE REVIEW

File: zdemo_abapgit_new.prog.abap [hunk 1]
```diff
@@ -4,3 +4,5 @@
 *&
 *&---------------------------------------------------------------------*
 REPORT ZDEMO_ABAPGIT_NEW.
+
+WRITE: 'Hello world'.
```
Comment (siangying) [inline zdemo_abapgit_new.prog.abap]: Put a comment to your code

Comment (siangying) [inline zdemo_abapgit_new.prog.abap]: Put a comment to your code

File: zdemo_abapgit_new.prog.abap [hunk 2]
```diff
@@ -4,3 +4,5 @@
 *&
 *&---------------------------------------------------------------------*
 REPORT ZDEMO_ABAPGIT_NEW.
+
```
Comment (siangying) [inline zdemo_abapgit_new.prog.abap]: Need to Add more logic

File: zdemo_abapgit_new.prog.abap (hunk 1 above)
Comment (siangying) [inline zdemo_abapgit_new.prog.abap]: Put a comment to your code

File: zdemo_abapgit_new.prog.abap [hunk 3]
```diff
@@ -4,3 +4,5 @@
...
+WRITE: 'Hello world'.  "++
```
Comment (sysim) [inline zdemo_abapgit_new.prog.abap]: Comment does not follow naming convention. Please change it.

Comment (sysim) [inline zdemo_abapgit_new.prog.abap]: Comment does not follow naming convention.

Comment (siangying) [inline zdemo_abapgit_new.prog.abap]: Comment does not follow naming convention. Please change

Comment (siangying) [inline zdemo_abapgit_new.prog.abap]: Comment does not follow naming convention. Please change

Comment (siangying) [inline zdemo_abapgit_new.prog.abap]: Follow naming convention

**Pull request closed without merging**